app/frontend/frontend.py

```diff
@@ -20,4 +23,18 @@ def feeds():
     users_subscriptions = current_user.subscribed
 
     return render_template('feeds.html',
-                            subscriptions=users_subscriptions)
\ No newline at end of file
+                            subscriptions=users_subscriptions)
+
+@bp.route('/feeds/<int:id>')
+@login_required
```

Comment:
Let's not make login required here so that users can browse our site without pestering them to login.